relayer/crates/starknet-cli/src/commands/start.rs

```diff
@@ -43,4 +43,10 @@ pub struct StartRelayerArgs {
 
     #[clap(long = "stop-after-blocks", required = false)]
     stop_after_blocks: Option<humantime::Duration>,
+
+    #[clap(long = "refresh-rate-a-to-b", required = false)]
+    refresh_rate_a: Option<humantime::Duration>,
+
+    #[clap(long = "refresh-rate-b-to-a", required = false)]
+    refresh_rate_b: Option<humantime::Duration>,
```

Comment:
```suggestion
    #[clap(long = "refresh-rate-a", required = false)]
    refresh_rate_a: Option<humantime::Duration>,

    #[clap(long = "refresh-rate-b", required = false)]
    refresh_rate_b: Option<humantime::Duration>,
```

actually do we even need the `long` attribute? it's the same as the field names.